Elaborate the release instruction (#357)
apache/openwhisk-release: docs/release_instructions.md

     - repository: URL of the repository
     - branch: git branch being released (`master` or a release branch name)

> **Important Note1:** If you are releasing a Node.js package which

Reviewer: ```suggestion
**Important Note 1:** If you are releasing a Node.js package which
```

contains a `package.json`, you should make sure the `version`
specified in `package.json` matches your release version. If you
also have a `package-lock.json` file, it too must have a matching
`version` property. See additional tips for [releasing npm
packages](#publishing-to-npm).

> **Important Note2:** If you are releasing the `openwhisk` or `openwhisk-deploy-kube` repositories, 

Reviewer: ```suggestion
**Important Note 2:** If you are releasing the `openwhisk` or `openwhisk-deploy-kube` repositories, 
```

the branch and commit hash in the release contents should be the release branch and the latest commit hash in the release branch.

> Please be especially noted that they are not from the `master` branch.

Reviewer: ```suggestion
Please be careful not to use the `master` branch.
```
